content/deploy-troubleshooting.md

```diff
@@ -4,36 +4,51 @@ order: 15
 
-A common cause of this error is that your `MONGO_URL` variable is missing or is set incorrectly. You can verify what `MONGO_URL` Galaxy is using by going to the app's dashboard and choosing the settings tab. To learn how to set it correctly, check the following resources:
+Note that code-level review lies outside the scope of Galaxy's support.
 
-* [How do I set Environment Variables?](https://galaxy.meteor.com/help/setting-environment-variables) will show you how to set up your `settings.json` file.
+<h2 id="503-errors">503 errors and Deployment failures</h2>
+
+The 503 error will show `Service Unavailable: No healthy endpoints to handle the request` when you try to visit your URL. This indicates that our image builder was not able to successfully create a container, when attempting to deploy. This usually indicates a problem in your code that prevents deployment.
+
+You may see the 503 error temporarily, if trying to resolve a URL in your browser within 1-2 minutes of issuing your deploy command. If you've waited 15 minutes or more, the problem will almost surely not resolve itself by waiting longer.
+
+Begin by checking the logs tab to see if your app is crashing. The 'Service' tab will show build errors. Most of the time, the key to a solution will be found in the exception or error messages. Keep iterating on code fixes and deployments until the error goes away, a new error appears, or your app deploys successfully.
+
+A common cause of the 503 error is that your `MONGO_URL` variable is missing or is set incorrectly. You can verify what `MONGO_URL` Galaxy is using by going to the app's dashboard and choosing the settings tab. To learn how to set it correctly, check the following resources:
+
+* [Environment variables](/environment-variables.html) will show you how to set up your `settings.json` file.
 * [This compose.io article](https://www.compose.io/articles/meteors-new-galaxy-and-the-perfectly-composed-companion/) explains the settings for `MONGO_URL` and `MONGO_OPLOG_URL` in detail.
 
 If you believe your `MONGO_URL` is set correctly, try the following:
 
-* Check the app dashboard and verify the app status is green (healthy)
-* Check the logs tab to see if the app is crashing. If so, look at the exception to determine the issue.
+* Check the app dashboard and verify the app status is green (healthy).
 * Run `dig +show [your app's domain]` in the terminal and verify that its CNAME points to `galaxy-ingress.meteor.com`. You can learn how to set up DNS [here](configuring-dns). If you recently changed your DNS settings, you may need to wait for the new records to propagate. DNS changes often propagate within 30 minutes (depending on the TTL configured for the record set), but in some cases it can take up to 24 hours. Contact your DNS provider if you think there is a problem.
```

Comment:
oops, this is out of date since it's not correct for EU

mention both, and probably include the longer version with us-east-1 in it so it's obvious why there are two.